apps/server/src/modules/fwu-learning-contents/fwu-learning-contents.module.ts

```diff
@@ -0,0 +1,70 @@
+import { Module, NotFoundException } from '@nestjs/common';
+import { S3Client } from '@aws-sdk/client-s3';
+import { AuthorizationModule } from '@src/modules/authorization';
+import { HttpModule } from '@nestjs/axios';
+import { CoreModule } from '@src/core';
+import { ConfigModule } from '@nestjs/config';
+import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
+import { AuthenticationModule } from '@src/modules/authentication/authentication.module';
+import { MikroOrmModule, MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs';
+import { Account, User, Role, School, System, SchoolYear } from '@shared/domain';
+import { createConfigModuleOptions, DB_PASSWORD, DB_URL, DB_USERNAME } from '@src/config';
+import { Logger } from '@src/core/logger';
+import { s3Config, config } from './fwu-learning-contents.config';
+import { FwuLearningContentsController } from './controller/fwu-learning-contents.controller';
+import { FwuLearningContentsUc } from './uc/fwu-learning-contents.uc';
+import { S3Config } from './interface/config';
+
+const providers = [
+	Logger,
+	{
+		provide: 'S3_Client',
```

Comment:
shouldn't this be named more special, like FWUS3 whatever?
since its the name that is the key for nest? not sure

guess it works now since new file service and this is not in the same module? if so sounds like bad practice to me